lib/ReactViews/Custom/Chart/ZoomX.jsx

```diff
@@ -20,9 +20,9 @@ class ZoomX extends React.Component {
     this.zoom = d3Zoom()
       .scaleExtent(props.scaleExtent)
       .translateExtent(props.translateExtent)
-      .on("zoom", () =>
-        props.onZoom(d3Event.transform.rescaleX(this.props.initialScale))
-      );
+      .on("zoom", (event) => {
+        props.onZoom(event.transform.rescaleX(this.props.initialScale))
+      });
```

Comment:
d3event is removed in latest d3-selection, instead the event is passed as argument for the zoom callback.